duckduckgo_search/duckduckgo_search_async.py

```diff
@@ -16,6 +16,7 @@
 from .models import MapsResult
 from .utils import _extract_vqd, _is_500_in_url, _normalize, _normalize_url, _random_browser, _text_extract_json
 
+logging.basicConfig()
```

Comment:
why?

Reply:
Otherwise, the warning I was trying to raise using logging.warning was not displayed in stdout. However, I removed the line because I used the warnings library instead as you suggested.